lib/CleantalkSP/Updater/UpdaterScripts.php

```diff
@@ -1008,4 +1008,12 @@ public static function updateTo_2_96_0() //phpcs:ignore PSR1.Methods.CamelCapsMe
         );
         $spbc->save('data');
     }
+
+    public static function updateTo_2_97_0() //phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    {
+        global $spbc;
+
+        $spbc->data['scanner']['first_scan__front_end'] = 1;
```

Comment:
Check if this flag `first_scan__front_end `not exist firstly and do changing it after.

Reply:
done